outputs.tf

```diff
@@ -33,6 +33,11 @@ output "eks_cluster_version" {
   value       = join("", aws_eks_cluster.default.*.version)
 }
 
+output "eks_cluster_identity_oidc_issuer" {
+  description = "The OIDC Identity issuer for the cluster"
+  value       = join("", aws_eks_cluster.default.*.identity.0.oidc.0.issuer)
```

Comment:
not sure this will work then `var.enabled = false`
https://github.com/cloudposse/terraform-aws-eks-cluster/blob/master/main.tf#L98

in this case `aws_eks_cluster.default.*` will be an empty list, and `aws_eks_cluster.default.*.identity` will probably be `null`, and `aws_eks_cluster.default.*.identity.0` could fail.

that's why for `certificate_authority_data` it was implemented like this https://github.com/cloudposse/terraform-aws-eks-cluster/blob/master/auth.tf#L30

although it was done in TF 0.11 first and then ported to TF 0.12.

need to test when `var.enabled = false`, or implement it similar to `certificate_authority_data`

Reply:
Thanks for the review! I've had a test with `var.enabled = false`, aka a count of `0` on the `aws_eks_cluster` resource. 

No apparent errors, and the output simply returns an empty string, similar to `eks_cluster_certificate_authority_data`. 

Perhaps there are changes in how this is handled between 0.11 and 0.12?

Reply:
Either way, I'm happy to update to follow the pattern of `certificate_authority_data` for consistency if you'd prefer?

Reply:
@benclapp thanks
if it works with TF 0.12, it's good, no need to change it.
`certificate_authority_data` was done for TF 0.11 and was just ported to 0.12 w/o changes